From: Qingqing <wangyiqing@kylinos.cn>
Date: Fri, 17 Jul 2026 09:50:04 +0000
Subject: Translated using Weblate (Arabic)

Currently translated at 100.0% (6 of 6 strings)

Translation: openkylin-huanghe/ukui-panel-calendar
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/ukui-panel-calendar/ar/
---
 .../ukui-panel-calendar/i18n/org.ukui.panel.calendar_ar.ts   | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/widgets/ukui-panel-calendar/i18n/org.ukui.panel.calendar_ar.ts b/widgets/ukui-panel-calendar/i18n/org.ukui.panel.calendar_ar.ts
index 34c51bb..cdfe97c 100644
--- a/widgets/ukui-panel-calendar/i18n/org.ukui.panel.calendar_ar.ts
+++ b/widgets/ukui-panel-calendar/i18n/org.ukui.panel.calendar_ar.ts
@@ -5,30 +5,30 @@
     <name>Calendar</name>
     <message>
         <source> new notification</source>
-        <translation type="unfinished"></translation>
+        <translation> إشعار جديد</translation>
     </message>
     <message>
         <source> new notifications</source>
-        <translation type="unfinished"></translation>
+        <translation> إشعار جديد</translation>
     </message>
     <message>
         <source>Notification center</source>
-        <translation type="unfinished"></translation>
+        <translation>مركز الإشعارات</translation>
     </message>
 </context>
 <context>
     <name>main</name>
     <message>
         <source>Set Time and Date</source>
-        <translation type="unfinished"></translation>
+        <translation>ضبط الوقت والتاريخ</translation>
     </message>
     <message>
         <source>Open</source>
-        <translation type="unfinished"></translation>
+        <translation>فتح</translation>
     </message>
     <message>
         <source>Set up notification center</source>
-        <translation type="unfinished"></translation>
+        <translation>إعداد مركز الإشعارات</translation>
     </message>
 </context>
 </TS>
